ci/filter_example_setup.sh

```diff
@@ -8,7 +8,10 @@ set -e
 ENVOY_FILTER_EXAMPLE_GITSHA="493e2e5bee10bbed1c3c097e09d83d7f672a9f2e"
 ENVOY_FILTER_EXAMPLE_SRCDIR="${BUILD_DIR}/envoy-filter-example"
 
-export ENVOY_FILTER_EXAMPLE_TESTS="//:echo2_integration_test //http-filter-example:http_filter_integration_test //:envoy_binary_test"
+export ENVOY_FILTER_EXAMPLE_TESTS=(
```

Comment:
This one too?

Reply:
ah yep - ill check the other files...

Reply:
i think this is the only one in this PR - we can check for other exported arrays after